httplib.h

```diff
@@ -524,8 +524,8 @@ struct Response {
   void set_header(const std::string &key, const std::string &val);
 
   void set_redirect(const std::string &url, int status = 302);
-  void set_content(const char *s, size_t n, const std::string &content_type);
-  void set_content(const std::string &s, const std::string &content_type);
+  void set_content(const char *s, size_t n, const std::string &content_type = {});
+  void set_content(const std::string &s, const std::string &content_type = {});
```

Comment:
Could you remove the default parameter `= {}` as well please?

Reply:
You don't have to use `set_content` method at all if you want to set the content text only. You can simply do it by `res.body = s` `with std::string` or `res.body.assign(s, n)` with `const char*`.

https://github.com/yhirose/cpp-httplib/blob/20a7f088cea4df658ed66b17153148b75e64881e/httplib.h#L5220